Library/Homebrew/cmd/update-report.rb

```diff
@@ -101,6 +104,32 @@ def install_core_tap_if_necessary
     ENV["HOMEBREW_UPDATE_BEFORE_HOMEBREW_HOMEBREW_CORE"] = revision
     ENV["HOMEBREW_UPDATE_AFTER_HOMEBREW_HOMEBREW_CORE"] = revision
   end
+
+  def migrate_legacy_cache_if_necessary
```

Comment:
If
1. ~/Library/Caches/Homebrew does not exist
2. /Library/Homebrew/Caches does exist
3. /Library/Homebrew/Caches passes a recursive ownership check
4. /Library/Homebrew/Caches passes a recursive permissions check
5. /Library/Homebrew/Caches passes a same volume check
6. migration has not already been attempted

then

```
mv /Library/Caches/Homebrew ~/Library/Caches/Homebrew
```
else do not pass Go, do not collect $200.